src/button/README.md

```diff
@@ -2,7 +2,48 @@
+| Primary   | ![Primary](/docs/assets/button-types-primary.png)     | Emphasizes the highest priority action in a view.                                  |
+| Secondary | ![Secondary](/docs/assets/button-types-secondary.png) | Used for additional actions in a view that already features a primary action.      |
+| Icon      | ![Icon](/docs/assets/button-types-icon.png)           | A space-efficient style that renders a single icon to represent a specific action. |
+
+### Best Practices
+
+| ❌ Don't                                                        | ✅ Do                                                              |
+| --------------------------------------------------------------- | ------------------------------------------------------------------ |
+| ![Image placeholder](/docs/assets/img-placeholder.png)          | ![Image placeholder](/docs/assets/img-placeholder.png)             |
+| Don't use multiple primary buttons unless absolutely necessary. | Provide a single primary button with one or more secondary actions |
+
+| ❌ Don't                                                            | ✅ Do                                                                                                                                             |
+| ------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ![Image placeholder](/docs/assets/img-placeholder.png)              | ![Image placeholder](/docs/assets/img-placeholder.png)                                                                                            |
+| Don't override button widths to always fill their parent container. | Allow buttons to fill a [VS Code View](https://code.visualstudio.com/api/references/extension-guidelines#views) until a maximum width is reached. |
+
+| ❌ Don't                                               | ✅ Do                                                  |
+| ------------------------------------------------------ | ------------------------------------------------------ |
+| ![Image placeholder](/docs/assets/img-placeholder.png) | ![Image placeholder](/docs/assets/img-placeholder.png) |
+| Don't use all caps or lowercase text.                  | Use sentence case for all button text.                 |
+
+| ❌ Don't                                               | ✅ Do                                                                                          |
+| ------------------------------------------------------ | ---------------------------------------------------------------------------------------------- |
+| ![Image placeholder](/docs/assets/img-placeholder.png) | ![Image placeholder](/docs/assets/img-placeholder.png)                                         |
```

Comment:
Don't know why but the second (rightmost) placeholder image is bigger than all the other images. Not super important for the placeholders but keep an eye out once you replace them with proper illustrations.

Reply:
I occasionally saw this issue locally when using a GH markdown extension. I can't seem to repro it again. Can you check again to make sure that its still happening?